fvtest/porttest/omrsockTest.cpp

```diff
@@ -1272,7 +1272,12 @@ TEST(PortSockTest, poll_functionality_many_sockets)
 	for (int32_t i = 0; i < 8; i++) {
 		ASSERT_EQ(OMRPORTLIB->sock_socket(OMRPORTLIB, &sockets[i], OMRSOCK_AF_INET, OMRSOCK_STREAM, OMRSOCK_IPPROTO_DEFAULT), 0);
 		ASSERT_EQ(OMRPORTLIB->sock_fcntl(OMRPORTLIB, sockets[i], OMRSOCK_O_NONBLOCK), 0);
+#if defined(OMR_OS_AIX)
+	/* OMRSOCK_POLLHUP not available on aix */
+		ASSERT_EQ(OMRPORTLIB->sock_pollfd_init(OMRPORTLIB, &pollArray[i], sockets[i], OMRSOCK_POLLIN), 0);
+#else /* defined(OMR_OS_AIX) */
 		ASSERT_EQ(OMRPORTLIB->sock_pollfd_init(OMRPORTLIB, &pollArray[i], sockets[i], OMRSOCK_POLLIN | OMRSOCK_POLLHUP), 0);
+#endif /* defined(OMR_OS_AIX) */
```

Comment:
Please indent consistently and correct the spelling of AIX. Also, this should not equate the availability of `OMRSOCK_POLLHUP` and `OMR_OS_AIX`; suggest instead:
```
#if defined(OMRSOCK_POLLHUP)
		ASSERT_EQ(OMRPORTLIB->sock_pollfd_init(OMRPORTLIB, &pollArray[i], sockets[i], OMRSOCK_POLLIN | OMRSOCK_POLLHUP), 0);
#else /* defined(OMRSOCK_POLLHUP) */
		/* OMRSOCK_POLLHUP not available on AIX */
		ASSERT_EQ(OMRPORTLIB->sock_pollfd_init(OMRPORTLIB, &pollArray[i], sockets[i], OMRSOCK_POLLIN), 0);
#endif /* defined(OMRSOCK_POLLHUP) */
```
Same comment for similar block beginning on line 1307.